src/hotspot/share/classfile/symbolTable.cpp

```diff
@@ -923,14 +923,14 @@ void SymbolTable::print_histogram() {
   tty->print_cr("  Total removed            " SIZE_FORMAT_W(7), _symbols_removed);
   if (_symbols_counted > 0) {
     tty->print_cr("  Percent removed          %3.2f",
-          ((float)_symbols_removed / _symbols_counted) * 100);
+          ((float)_symbols_removed / (float)_symbols_counted) * 100);
```

Comment:
I think you can also just cast once here.

Reply:
These should probably be double rather than float - the float result will be promoted to double to be passed as an argument anyway.

Reply:
Changed to double.